From b039705e6fb633dbdf67dc85fa8a58f6a69a33cd Mon Sep 17 00:00:00 2001
From: Fugang Duan <B38611@freescale.com>
Date: Sat, 19 May 2012 11:01:54 +0800
Subject: [PATCH 1514/2463] ENGR00210075-2 - MX6SL MSL: Add SPDC in
 imx6s_defconfig

 Add Sipix panel option in imx6s_defconfig file

 Signed-off-by: Fugang Duan  <B38611@freescale.com>
---
 arch/arm/configs/imx6s_defconfig |    3 +++
 1 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/arch/arm/configs/imx6s_defconfig b/arch/arm/configs/imx6s_defconfig
index 1dbe91b..3db0aee 100644
--- a/arch/arm/configs/imx6s_defconfig
+++ b/arch/arm/configs/imx6s_defconfig
@@ -275,6 +275,7 @@ CONFIG_IMX_HAVE_PLATFORM_IMX_VIIM=y
 CONFIG_IMX_HAVE_PLATFORM_PERFMON=y
 CONFIG_IMX_HAVE_PLATFORM_LDB=y
 CONFIG_IMX_HAVE_PLATFORM_IMX_PXP=y
+CONFIG_IMX_HAVE_PLATFORM_IMX_SPDC=y
 CONFIG_IMX_HAVE_PLATFORM_IMX_EPDC=y
 CONFIG_IMX_HAVE_PLATFORM_IMX_SPDIF=y
 CONFIG_IMX_HAVE_PLATFORM_VIV_GPU=y
@@ -1726,6 +1727,8 @@ CONFIG_FB_MXC_TRULY_WVGA_SYNC_PANEL=y
 CONFIG_FB_MXC_EINK_PANEL=y
 # CONFIG_FB_MXC_EINK_AUTO_UPDATE_MODE is not set
 # CONFIG_FB_MXC_ELCDIF_FB is not set
+# CONFIG_FB_MXC_SIPIX_PANEL is not set
+# CONFIG_FB_MXC_SIPIX_AUTO_UPDATE_MODE is not set
 CONFIG_FB_MXC_HDMI=y
 
 #
-- 
1.7.7.4

